docs/25_developer.md

```diff
@@ -61,18 +59,19 @@ Developer Tools
 ---------------
 
 We are using many of the following tools. Developers need not download
-and install this long list of technologies. We will provide developers
-a pre-configured Eclipse install, a bootstrapping Eclipse workspace,
+and install this long list of technologies. We have provided developers
+with a pre-configured [Eclipse](https://eclipse.org/) install,
+a bootstrapping Eclipse workspace,
 and a VM image which can be loaded into VirtualBox or other comparable
 virtualization platform. We provide these resources in order to both
 decrease new developers' ramp-up time as well as to standardize on
 specific versions of tools for development.
 
-We expect to provide the initial Eclipse snapshot and workspace in the
-week of 9 July 2017.
+Instructions for installing Eclipse and the plugins we leverage
+are at [README-ECLIPSE.md](../server/README-ECLIPSE.md).
 
 * [GitHub](https://github.com/FreeAndFair/ColoradoRLA) for distributed
-  version control, change tracking, and development documentation
+  version control, issue tracking, and development documentation
```

Comment:
Issues and changes are not the same thing, but I think change tracking is inherent in version control, so it all works out.